src/components/JobsPanelParameters/jobsPanelParameters.util.js

```diff
@@ -1,27 +1,33 @@
+export const convertParamValue = (value, type) =>
+  ['int', 'float', 'number'].includes(type) && Number.isFinite(Number(value))
```

Comment:
Using `Number.isFinite` and not `!Number.isNaN`, because I don't want to only rule out `NaN`, but also `Infinity`. If the string entered to the text box is `"infinity"` then `Number("Infinity")` would give the number `Infinity` value, which is not `NaN`. Similarly to `NaN`, `Infinity` does not exist in JSON would end up showing as `null` or another  unexpected value.